### PR TITLE
Add OpenUV docs that describe the 403/re-auth idiosyncrasy

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

This PR updates the OpenUV docs to outline a re-auth idiosyncrasy.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85573
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Thanks, @bachya 👍

File: source/_integrations/openuv.markdown [hunk 1]
```diff
@@ -174,3 +174,21 @@ automation:
...
+
+1. An invalid API key
+2. An API key whose daily/monthly limit is reached
+
+Unfortunately, the integration is unable to determine which is which from the API data
+provided by OpenUV. So, this strategy is followed:
+
+1. Any `HTTP 403` will trigger a re-auth flow.
+2. In the case of an overrun API call limit, once the limit expires and the coordinator
+   successfully retrieves data, any existing re-auth flow is canceled.
```
Comment (frenck) [inline source/_integrations/openuv.markdown]: Hmm, second thought. "re-auth flow" and "coordinator" are pretty much backend/developer terms. An average user won't have any idea what those mean. 

Can we generalize those to make it more clear what is going on?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/openuv.markdown (hunk 1 above)
Comment (bachya) [reply source/_integrations/openuv.markdown]: 10-4. I'll think of something more user-friendly.

Comment (bachya) [reply source/_integrations/openuv.markdown]: Thoughts on https://github.com/home-assistant/home-assistant.io/pull/25671/commits/3a70d64b58d2357f1728be11b740298ac9e8a5dc?

File: source/_integrations/openuv.markdown [hunk 2]
```diff
@@ -174,3 +174,23 @@ automation:
...
+
+1. An invalid API key
+2. An API key whose daily/monthly limit is reached
+
+Unfortunately, the integration is unable to determine which is which from the API data
+provided by OpenUV. So, this strategy is followed:
+
+1. Any `HTTP 403` response will create a persistent notification asking you to
+   re-authenticate the OpenUV integration.
+2. In the case of an overrun API call limit, once the the `homeassistant.update_entity`
```
Comment (bachya) [inline source/_integrations/openuv.markdown]:
```suggestion
2. In the case of an overrun API call limit, once the `homeassistant.update_entity`
```

File: source/_integrations/openuv.markdown [hunk 3]
```diff
@@ -174,3 +174,23 @@ automation:
...
+1. An invalid API key
+2. An API key whose daily/monthly limit is reached
+
+Unfortunately, the integration is unable to determine which is which from the API data
+provided by OpenUV. So, this strategy is followed:
+
+1. Any `HTTP 403` response will create a persistent notification asking you to
+   re-authenticate the OpenUV integration.
+2. In the case of an overrun API call limit, once the `homeassistant.update_entity`
+   service call is successful, existing re-authentication notifications will
```
Comment (bachya) [inline source/_integrations/openuv.markdown]:
```suggestion
   service call is again successful, existing re-authentication notifications will
```

**Review by frenck — APPROVED** [review]
Thanks, @bachya 👍

**Pull request merged**